packages/core/demo/data/bubble.ts

```diff
@@ -1,189 +1,94 @@
-	datasets: [
-		{
-			label: "Dataset 1",
-			data: [
-				{
-					date: new Date(2019, 0, 1),
-					value: 10000,
-					radius: 6
-				},
-				{
-					date: new Date(2019, 0, 5),
-					value: 45000,
-					radius: 5
-				},
-				{
-					date: new Date(2019, 0, 8),
-					value: 10000,
-					radius: 8
-				},
-				{
-					date: new Date(2019, 0, 13),
-					value: 49213,
-					radius: 11
-				},
-				{
-					date: new Date(2019, 0, 17),
-					value: 51213,
-					radius: 3
-				}
-			]
-		},
-		{
-			label: "Dataset 2",
-			data: [
-				{
-					date: new Date(2019, 0, 2),
-					value: 12000,
-					radius: 4
-				},
-				{
-					date: new Date(2019, 0, 6),
-					value: 57312,
-					radius: 6
-				},
-				{
-					date: new Date(2019, 0, 8),
-					value: 21432,
-					radius: 8
-				},
-				{
-					date: new Date(2019, 0, 15),
-					value: 70323,
-					radius: 3
-				},
-				{
-					date: new Date(2019, 0, 19),
-					value: 21300,
-					radius: 5
-				}
-			]
+import { lineTimeSeriesData } from "./line";
+
+export const bubbleLinearData = [
+	{ group: "Dataset 1", sales: 10000, profit: 32100, surplus: 50000 },
+	{ group: "Dataset 1", sales: 12000, profit: 23500, surplus: 34000 },
+	{ group: "Dataset 1", sales: 14000, profit: 53100, surplus: 63000 },
+	{ group: "Dataset 1", sales: 15000, profit: 42300, surplus: 43000 },
+	{ group: "Dataset 1", sales: 16000, profit: 12300, surplus: 55000 },
+	{ group: "Dataset 2", sales: 11000, profit: 12400, surplus: 25000 },
+	{ group: "Dataset 2", sales: 13000, profit: 34500, surplus: 35000 },
+	{ group: "Dataset 2", sales: 13500, profit: 23100, surplus: 55000 },
+	{ group: "Dataset 2", sales: 15500, profit: 63200, surplus: 35000 },
+	{ group: "Dataset 2", sales: 15750, profit: 24300, surplus: 64000 }
+];
+
+export const bubbleLinearOptions = {
+	title: "Bubble (linear)",
+	axes: {
+		bottom: {
+			title: "Sales",
+			identifier: "sales",
+			includeZero: false
 		},
-		{
-			label: "Dataset 3",
-			data: [
-				{
-					date: new Date(2019, 0, 1),
-					value: 50000,
-					radius: 5
-				},
-				{
-					date: new Date(2019, 0, 1),
-					value: 15000,
-					radius: 3
-				},
-				{
-					date: new Date(2019, 0, 2),
-					value: 20000,
-					radius: 8
-				},
-				{
-					date: new Date(2019, 0, 2),
-					value: 39213,
-					radius: 4
-				},
-				{
-					date: new Date(2019, 0, 3),
-					value: 61213,
-					radius: 3
-				},
-				{
-					date: new Date(2019, 0, 3),
-					value: 50000,
-					radius: 5
-				},
-				{
-					date: new Date(2019, 0, 4),
-					value: 15000,
-					radius: 3
-				},
-				{
-					date: new Date(2019, 0, 4),
-					value: 20000,
-					radius: 2
-				},
-				{
-					date: new Date(2019, 0, 5),
-					value: 39213,
-					radius: 4
-				},
-				{
-					date: new Date(2019, 0, 6),
-					value: 61213,
-					radius: 3
-				}
-			]
+		left: {
+			title: "Profit",
+			identifier: "profit",
+			includeZero: false
+		}
+	},
+	bubble: {
+		radiusIdentifier: "surplus"
```

Comment:
no

Reply:
so let's name it just `identifier`?

Reply:
hi, I think this `radius` could be thought as a theoretical new Axis, because it's like a "third dimension". We could keep the same shape of the other `top, left, right, bottom` axes, so:
```
bubble: {
  radius: {
    identifier: "xxx",
    scaleType: "linear" // does not make much sense to use different scaleTypes here, but we could support it anyway and the choice can be left to the user
    // ...
  }
}
```

I think the main advantage here would be the future extensibility. For example, that would be the place to change the limits of the scale, or some legend configurations.

Reply:
That makes a lot of sense actually.

Reply:
probably should be `radiusMapsTo` now 👍